docs/dqx/docs/reference/quality_rules.mdx

```diff
@@ -38,6 +38,7 @@ You can also define your own custom checks (see [Creating custom checks](#creati
 | `sql_expression`                   | Checks whether the values meet the condition provided as an SQL expression, e.g. `a = 'str1' and a > b`. SQL expressions are evaluated at runtime, so ensure that the expression is safe and that functions used within it (e.g. h3_ischildof, division) do not throw exceptions. You can achieve this by validating input arguments or columns beforehand using guards such as CASE WHEN, IS NOT NULL, RLIKE, or type try casts. | `expression`: sql expression to check on a DataFrame (fail the check if expression evaluates to True, pass if it evaluates to False); `msg`: optional message to output; `name`: optional name of the resulting column (it can be overwritten by `name` specified at the check level); `negate`: if the condition should be negated; `columns`: optional list of columns to be used for reporting and as name prefix if name not provided, unused in the actual logic                  |
 | PII detection                      | Checks whether the values in the input column contain Personally Identifiable Information (PII). This check is not included in DQX built-in rules to avoid introducing 3rd-party dependencies. An example implementation can be found [here](/docs/reference/quality_rules#detecting-personally-identifiable-information-pii-using-a-python-function).                                                                            |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `is_data_fresh`                    | Checks whether the values in the input timestamp column are not older than the specified number of minutes from the base timestamp column. This is useful for identifying stale data due to delayed pipelines and helps catch upstream issues early.                                                                                                                                                                              | `column`: column of type timestamp/date to check (can be a string column name or a column expression); `max_age_minutes`: maximum age in minutes before data is considered stale; `base_timestamp`: optional base timestamp column from which the stale check is calculated, if not provided current_timestamp() function is used                                                                                                                                                      |
```

Comment:
Please also add this to the examples using yaml and dqx classes
https://github.com/databrickslabs/dqx/blob/main/docs/dqx/docs/reference/quality_rules.mdx?plain=1#L57
https://github.com/databrickslabs/dqx/blob/main/docs/dqx/docs/reference/quality_rules.mdx?plain=1#L353